### PR TITLE
Ch/home button

### PR DESCRIPTION
**Pull request by CaliHam**
### Description
Creates Home Button functionality.

### Testing
When a user clicks the “Home” button:

- we only see the Home section
- the home button is hidden
- the “Show New Random Cover” and “Save Cover” buttons are be visible again

### CODE REVIEW

**Comment by CaliHam** [comment]
After we merge the other two pull requests, we can double check the Home Button works from both "Make Your Own Cover" and "View Saved Covers" buttons.

File: src/main.js [hunk 1]
```diff
@@ -10,10 +10,7 @@ var saveCoverButton = document.querySelector('.save-cover-button');
 var viewSavedCoversButton = document.querySelector('.view-saved-button');
 var homeView = document.querySelector('.home-view');
 var savedView = document.querySelector('.saved-view');
-
 var makeNewButton = document.querySelector('.make-new-button');
-var homeButton = document.querySelector('.home-button');
-var saveCoverButton = document.querySelector('.save-cover-button');
 var homeView = document.querySelector('.home-view');
```
Comment (mbenfowler) [inline src/main.js:14]: looks like this one can go too; repeat of line11

File: src/main.js [hunk 2]
```diff
@@ -22,6 +19,7 @@ var savedCovers = [
   createCover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover;
+var currentView;
```
Comment (mbenfowler) [inline src/main.js:22]: thank youuuu!

**Review by mbenfowler — APPROVED** [review]
Works as expected!  We can fix the one extra variable in a follow-up!

**Pull request merged**